src/test/java/org/broadinstitute/dsm/util/model/Kit.java

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.dsm.util.model;
```

Comment:
If that Kit class is only used for the migration tool I wonder if we should move that somewhere else than just `util/model`…